folding/utils/runandlog.py

```diff
@@ -0,0 +1,142 @@
+import subprocess
+import tqdm
+from typing import List, Dict
+from collections import defaultdict
+import bittensor as bt 
+import copy
+import wandb
+from loguru import logger
+from dataclasses import asdict, dataclass
+
+import folding
+
+
+"""
+We need something thet gets returned (result dict) 
+We don't want it to grow each time its called
+We want it to be independent of gromacs
+We dont need to be worried about pdb_id 
+This needs to log to the event dict 
+"""
+
+
+def output_dict():
+    return defaultdict(str)
+
+class CommandFailedException(Exception):
+    pass
+
+class RunAndLog():
+    def __init__(self):
+        self.command_dict = defaultdict(output_dict)
+    
+    def run_cmd_commands(
```

Comment:
we can probably finally rename this class to just `run_bash_commands` or `run_commands`. `run_cmd_commands` is kinda dumb hahaha

Reply:
https://github.com/macrocosm-os/folding/pull/147/commits/1ab372af091e2b9fd1024788da1b2073243a1317